aws_lambda_builders/workflows/nodejs_npm/actions.py

```diff
@@ -151,7 +159,11 @@ def execute(self):
         try:
             LOG.debug("NODEJS installing ci in: %s", self.install_dir)
 
-            self.subprocess_npm.run(["ci"], cwd=self.install_dir)
+            command = ["ci"]
+            if self.install_links:
+                command.append("--install-links")
+
+            self.subprocess_npm.run(command, cwd=self.install_dir)
```

Comment:
I've realized the CI action maybe should've just been a subclass of the normal install action, and in general I'm not sure why we don't use the same flags. But it might be too late to change now.